src-tauri/tests/fixtures/bundler/import_jsx/input/utils.jsx

```diff
@@ -0,0 +1,10 @@
+const React = window.__DESKULPT__.defaultDeps.React;
+
+export default function getMessage(){
+  return(
+    <div>
+      <p>This React component is successfully imported.</p>
+    </div>
+  );
+}
```

Comment:
```suggestion
export default function MessageInner() {
  return <div>This React component is successfully imported.</div>;
}
```

NIT code style, and please follow the convention that React functional components are capitalized.